src/actionbar/index.css

```diff
@@ -0,0 +1,53 @@
+@import '../commons/index.css';
+
+:root {
+  --spectrum-actionbar-height: var(--spectrum-global-dimension-size-600);
+  --spectrum-actionbar-padding-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-margin-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-offset-y: var(--spectrum-global-dimension-size-200);
+
+  --spectrum-actionbar-min-width: 280px;
+  --spectrum-actionbar-max-width: 1280px;
```

Comment:
This seems a little arbitrary. Where does this value come from?

Reply:
It is from Lindsay's XD file. https://precursors.s3-sj1.corp.adobe.com/files/action-bar.xd
I am thinking moving it to dna, but don't how.

Reply:
min-width and max-width are from here: 
![image](https://user-images.githubusercontent.com/1207520/57488511-29810f80-7268-11e9-90ea-594c9ad40938.png)

Reply:
Ok excellent, as long as it has a source. I will help you get a PR into DNA after this is in and we can hardcode for now.